tests/nightly/test_large_array.py

```diff
@@ -742,7 +761,7 @@ def test_activation():
     # Hyperbolic tangent (tanh)
     # y = (exp(x)-exp(-x))/(exp(x)+exp(-x))
     a = mx.nd.Activation(a, act_type="tanh")
-    tanh_x = (np.exp(-2)-np.exp(2))/(np.exp(-2)+np.exp(2))
+    tanh_x = (np.exp(test_x)-np.exp(-test_x))/(np.exp(test_x)+np.exp(-test_x))
```

Comment:
nit: add space around operators. Do this across the entire file.

Reply:
PEP8 / pylint both didn't give error for this. I'll do it but is there some other linting tool apart from PEP8 and pylint?

If it's google style guide tool?
Is this the way to go?
https://stackoverflow.com/questions/29597618/is-there-a-tool-to-lint-python-based-on-the-google-style-guide